temba/flows/views.py

```diff
@@ -1154,9 +1280,68 @@ def derive_title(self):
         def get_template_names(self):
             return "flows/flow_editor_next.haml"
 
+        def post(self, request, *args, **kwargs):
```

Comment:
As we already have the same function defined some code lines above, it would be great to have an inheritance on the function, if needed you could use class attributes for some parameters like `flows.flow_pdf_export_next` that's different from another function.